tox.ini

```diff
@@ -61,18 +61,21 @@ deps =
 setenv =
     PYTHONPATH =
 
-[testenv:coveralls]
-basepython = {[testenv:cover]basepython}
-commands =
-    pip install gcloud[grpc]
-    {[testenv]covercmd}
-    coveralls
+[testenv:codecov]
+basepython =
+    python2.7
```

Comment:
Ditto here on being clear to be the same env as cover: `basepython = {[testenv:cover]basepython}`

I'm just copy-pasting what was in `coveralls` for this one